web-console/future-openshift/toolbars/toolbars.md

```diff
@@ -0,0 +1,48 @@
+# Toolbars
+
+There are various toolbars used across the OpenShift console.
+
+
+## Basic toolbars
+* The most basic toolbars in OpenShift have a 'Create' button, a textbox filter, and an item count.
+
+Example
+![deployments](img/deployments-today.png)
```

Comment:
Call me crazy here if I'm wrong, but are there two different blues being used for the text on the Status and the Owner columns? If so, was that intentional?

Reply:
Nope, not on purpose! Updating now :)